pkg/util/jsonpath/jsonpath.go

```diff
@@ -234,6 +234,16 @@ func (j *JSONPath) evalArray(input []reflect.Value, node *ArrayNode) ([]reflect.
 			params[1].Value += value.Len()
```

Comment:
this can still result in a negative index (as can line 227)... need bounds checking in both directions below